develop.html.mako

```diff
@@ -27,22 +27,40 @@ on getting started.
 <a name="developers"></a>
 <h2>Development Community</h2>
 
-<p>Where do you find the developers ?   Three places:</p>
+<p>Where do you find the developers ?  Four places:</p>
 
 <ul>
-    <li>The <a href="/support.html#mailinglist">main
-    mailing list</a> is the most common - core devs assist users with issues of all kinds.</li>
-    <li>Developer discussion takes place
-    on the <a href="http://groups.google.com/group/sqlalchemy-devel">development mailing list</a>,
-    also hosted on Google groups.  Anyone can join this
-    list, though we ask that discussion be limited to architectural and developmental issues only.
-    </li>
-    <li>Some development discussion occurs on the
-    <code>#sqlalchemy-devel</code> IRC channel on Freenode.
-    This channel is <strong>only</strong> for users that are interested in contributing code,
-    tests, documentation, or other developmental resources, and no support issues will be
-    addressed here.   IRC-based support is available on the
-    <a href="/support.html#irc">main IRC channel</a>.</li>
+    <li>The <a href="/support.html#mailinglist">main mailing list</a> is the most 
+        common - core devs assist users with issues of all kinds.
+        </li>
+    <li>Developer discussion takes place on the 
+        <a href="http://groups.google.com/group/sqlalchemy-devel">development
+        mailing list</a>, also hosted on Google groups.  Anyone can join this list, 
+        though we ask that discussion be limited to architectural and developmental 
+        issues only.
+        </li>
+    <li>Some development discussion occurs on the <code>#sqlalchemy-devel</code>
+        IRC channel on Freenode. This channel is <strong>only</strong> for users 
+        that are interested in contributing code, tests, documentation, or other 
+        developmental resources.
+        <strong>No support issues will be addressed here.</strong>
+        IRC-based support is available on the <a href="/support.html#irc">main 
+        IRC channel</a>.
+        </li>
+    <li>A real-time virtual meeting of core developers and contributers occurs in the
```

Comment:
this is good to talk about the meeting, but there hsould be a separate paragraph that just discusses gitter as where development discussion occurs.